bv-java-grupa-1/iohomework/dataprocessing/ProcessCountryData.java

```diff
@@ -0,0 +1,108 @@
+package dataprocessing;
+
+import customexception.CountryNotFoundException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
+
+/**
+ * Deals with all the data processing from the online document.
+ */
+public class ProcessCountryData {
+    // Stores a list with country objects.
+    private List<CountryData> countryList = new ArrayList<>();
+
+    public ProcessCountryData() {
+        addCountries();
+    }
+
+    /**
+     * @return a list with every line from the online document.
+     */
+    public List<String> getURLData() {
```

Comment:
I think this method should be private; it's not useful to be called from outside this class.